model/atmosphere/diffusion/src/icon4py/model/atmosphere/diffusion/diffusion.py

```diff
@@ -461,28 +455,14 @@ def __init__(
                 offset_provider=self._grid.connectivities,
             )
         )
-        self.calculate_nabla2_for_theta = calculate_nabla2_for_theta.with_backend(
+        self.apply_diffusion_to_theta_and_exner = apply_diffusion_to_theta_and_exner.with_backend(
             self._backend
         ).compile(
             enable_jit=False,
```

Comment:
```suggestion
            enable_jit=False,
            apply_zdiffusion_t=self.config.apply_zdiffusion_t,
```
I think this should be a compile time option